project-progression-policy.md

```diff
@@ -72,8 +74,8 @@ the CCC requires that any trademarks be transferred to, or be available for
 use by, LF Projects, LLC.  Even if no trademarks exist, the current CCC
 policy is that projects be submitted to LF Projects, LLC.
 
-This submission entails sending the proposed Technical Charter (as reviewed
-by the TAC), and a signed [Project Contribution Agreement](https://lists.confidentialcomputing.io/g/main/files/TAC/Project%20Submissions/LF%20Projects%20--%20Form%20of%20Trademark%20and%20Account%20Assignment.docx).
+This submission entails sending the proposed Technical Charter (as reviewed by the TAC), 
+and a signed Project Contribution Agreement that was generated by the LF Project Formation Intake Form listed above.
```

Comment:
has Scott Nicolas verified that the generated agreement matches what is expected?